src/platform/forms-system/src/js/patterns/array-builder/ArrayBuilderSummaryPage.jsx

```diff
@@ -419,6 +425,11 @@ export default function ArrayBuilderSummaryPage({
       uiSchema['ui:title'] = (
         <>
           <Title textType="summaryTitle" />
+          {showSummaryDescription && (
+            <span className="vads-u-font-family--sans vads-u-font-weight--normal vads-u-display--block">
+              {getText('summaryDescription')}
```

Comment:
is the intention that this is only displayed in the "with items / with cards" use case? vs line 445 which is without?

Reply:
Yes, exactly.  This addition is covering the `with items / with cards` use case, versus the recently added `summaryDescriptionWithoutItems` which handles the yes/no UI screen, but disappears once items have been added.